.github/workflows/test.yaml

```diff
@@ -1,6 +1,5 @@
 # This file is part of Edgehog.
-#
```

Comment:
I think is better to remove this file and create a new one

Reply:
This is a new file git just thinks it was moved from fmt to test couldn't